### PR TITLE
Fix tests post-js-sdk-filters change

### PR DESCRIPTION
**Pull request by t3chguy**
Signed-off-by: Michael Telatynski <7t3chguy@gmail.com>

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
+    "resolveJsonModule": true,
```
Comment (turt2live) [inline tsconfig.json:5]: This seems unrelated.

Comment (t3chguy) [reply tsconfig.json:5]: > snook in a tsconfig change which I need for

as per RWI

Comment (turt2live) [reply tsconfig.json:5]: Please put it in its own PR so there's a bit more context for it, such as what it actually fixes - it's not obviously clear here.
